metric-schema-lang/src/main/java/com/palantir/metric/schema/lang/Validator.java

```diff
@@ -35,7 +42,7 @@ static void validate(MetricSchema schema) {
         schema.getNamespaces().forEach(Validator::validateNamespace);
     }
 
-    private static void validateNamespace(String namespace, MetricNamespace namespaceValue) {
+    private static void validateNamespace(String namespace, com.palantir.metric.schema.MetricNamespace namespaceValue) {
```

Comment:
I'm surprised that the `UnnecessarilyQualified` errorprone check isn't complaining about `com.palantir.metric.schema.MetricNamespace` when we have an import on line 22.

Reply:
odd, it might be because there is another class with the same name in the same package. Fixed it anyways

Reply:
It may be cleaner to work with the code if we prefix or suffix the lang classes in some way e.g. `LangMetricNamespace`.